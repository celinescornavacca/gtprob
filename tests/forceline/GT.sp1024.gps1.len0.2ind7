(((((((((VGA0,(AKA0,(JNA0,((VIB0,WIB0),DQA0)))),(((LCA0,((FPA0,GPA0),UEA0)),(((NHA0,((WYA0,XYA0),ISA0)),(HJB0,IJB0)),(NCA0,((TKA0,((YYA0,ZYA0),HQA0)),((EMA0,FMA0),PKA0))))),((((((JKA0,(RFB0,SFB0)),(IIB0,JIB0)),(FDB0,GDB0)),(((AEA0,BEA0),((JLB0,KLB0),((SLA0,TLA0),NIA0))),(((ENB0,FNB0),WDB0),(((USA0,VSA0),KSA0),TRA0)))),((PZA0,QZA0),FGA0)),((DHA0,EHA0),SBA0)))),(OJB0,PJB0)),(((RHA0,((QJB0,RJB0),EQA0)),((ETA0,FTA0),NGA0)),(((KIA0,(TDB0,UDB0)),((PBB0,QBB0),TWA0)),((BJB0,CJB0),(((XCB0,YCB0),(QMB0,RMB0)),OEA0))))),((((OCA0,((GHA0,(((MJB0,NJB0),WTA0),(DJB0,EJB0))),((CGB0,DGB0),(EHB0,FHB0)))),((RSA0,SSA0),(EJA0,(RHB0,SHB0)))),((((VEB0,WEB0),(((UWA0,VWA0),VOA0),DJA0)),((YLB0,ZLB0),JJB0)),(TCB0,UCB0))),((DAA0,(IHB0,JHB0)),((((FAA0,((((((ROA0,SOA0),(YSA0,ZSA0)),(OJA0,PJA0)),DFA0),(WLA0,XLA0)),((KIB0,LIB0),(LFA0,MFA0)))),((IMB0,JMB0),((BMA0,CMA0),(NDB0,(HLB0,ILB0))))),(((TBA0,UBA0),((QAB0,RAB0),(NNA0,ONA0))),((WDA0,(GGA0,(DOA0,(INB0,JNB0)))),((ICB0,JCB0),(((YJA0,ZJA0),(GMA0,HMA0)),(GOA0,(NVA0,OVA0))))))),((((((QBA0,((((ZXA0,AYA0),((DLA0,(QXA0,RXA0)),KKA0)),YDA0),JCA0)),(WNA0,XNA0)),(((((IDB0,JDB0),(PVA0,QVA0)),HHA0),(((QIB0,RIB0),FUA0),WIA0)),(((NTA0,OTA0),(STA0,TTA0)),((((ZBB0,ACB0),(OAB0,PAB0)),ZPA0),MPA0)))),(ODA0,((SJA0,TJA0),(YWA0,ZWA0)))),((((SAB0,TAB0),(PUA0,QUA0)),(IFA0,JFA0)),((WHA0,XHA0),(OHA0,(GRA0,HRA0))))),((EIB0,FIB0),(((((((OMB0,PMB0),ARA0),(HIA0,(ZTA0,((XDB0,YDB0),MZA0)))),(TFB0,(AGB0,BGB0))),UCA0),(BCA0,(SUA0,TUA0))),(CDA0,(RCB0,SCB0))))))))),(((GKA0,HKA0),(((OAA0,(SAA0,(JEA0,KEA0))),((((AGA0,BGA0),(((EDA0,FDA0),(LEA0,MEA0)),(BBA0,((AQA0,(PSA0,QSA0)),(GNB0,HNB0))))),(((RIA0,SIA0),((((RRA0,SRA0),(JHA0,((VGB0,(EMB0,FMB0)),(((ZCB0,ADB0),NYA0),DXA0)))),ZCA0),(((TVA0,(YJB0,ZJB0)),OOA0),(((HFB0,IFB0),LYA0),(UYA0,VYA0))))),((EZA0,FZA0),(FFB0,GFB0)))),(((VBA0,(BIA0,CIA0)),((FEB0,(ULB0,VLB0)),(((GKB0,HKB0),(XTA0,YTA0)),(WEA0,((DMA0,(CTA0,DTA0)),FFA0))))),(FIA0,GIA0)))),(((BXA0,CXA0),((MAB0,(VHB0,WHB0)),NJA0)),(((((BYA0,CYA0),BFA0),(KZA0,LZA0)),(((SYA0,TYA0),FYA0),NDA0)),GCA0)))),((AAA0,((((((IZA0,JZA0),(GZA0,HZA0)),(((((OLB0,PLB0),AUA0),((LDB0,MDB0),QWA0)),NSA0),PBA0)),((MOA0,NOA0),XCA0)),((((LXA0,(RBB0,SBB0)),YQA0),HDA0),((((CHB0,DHB0),CNA0),((AVA0,BVA0),(ALB0,BLB0))),XBA0))),(((MVA0,(CZA0,DZA0)),(ENA0,(ABB0,BBB0))),IAA0))),((((YBA0,(WLB0,XLB0)),((((WAB0,XAB0),(ATA0,BTA0)),((BQA0,CQA0),XDA0)),VCA0)),((QAA0,(((RDA0,SDA0),MCA0),NBA0)),(EWA0,(KBB0,LBB0)))),(((((((((AWA0,BWA0),XSA0),((TZA0,UZA0),RVA0)),(GNA0,(AZA0,BZA0))),MDA0),(FBA0,((EKB0,FKB0),(GGB0,(OIB0,PIB0))))),((HAA0,(UJA0,(ASA0,BSA0))),GAA0)),(((((PAA0,((REB0,SEB0),(DDB0,EDB0))),NAA0),((IQA0,JQA0),((ZBA0,((((HOA0,(JYA0,KYA0)),QMA0),(KVA0,(JGB0,KGB0))),(((UKA0,VKA0),ZEA0),(LIA0,MIA0)))),RBA0))),(WBA0,(PIA0,QIA0))),((((((TXA0,((AMB0,BMB0),CLB0)),(THB0,UHB0)),((UOA0,(CNB0,DNB0)),VLA0)),(JJA0,KJA0)),((((LCB0,MCB0),(ZDB0,AEB0)),(GMB0,HMB0)),(UXA0,VXA0))),(TIA0,(LQA0,MQA0))))),((((OGA0,PGA0),(YCA0,(ZDA0,(QEA0,(BFB0,CFB0))))),((MEB0,NEB0),(((TMA0,UMA0),(KRA0,(JXA0,KXA0))),(MKA0,NKA0)))),((((IEB0,JEB0),(ZEB0,AFB0)),(ZZA0,AAB0)),(ZLA0,AMA0)))))))),((((((((IBB0,JBB0),((IKA0,(UMB0,VMB0)),(QJA0,((WFB0,XFB0),LNA0)))),((((EYA0,(MKB0,NKB0)),((FWA0,GWA0),QRA0)),XIA0),(CRA0,DRA0))),((((MYA0,(AIB0,BIB0)),LVA0),(KUA0,LUA0)),((POA0,QOA0),(NQA0,OQA0)))),BAA0),((XJA0,(OSA0,(FVA0,GVA0))),((EEA0,(IWA0,JWA0)),(((WGA0,(SHA0,(PTA0,(XWA0,(TEB0,UEB0))))),(GBA0,(((((NZA0,OZA0),MUA0),(((QKA0,(UUA0,(GEB0,HEB0))),IGA0),(CMB0,DMB0))),(FEA0,(((SIB0,TIB0),GTA0),((BUA0,CUA0),CJA0)))),(YGA0,ZGA0)))),(((OXA0,PXA0),BKA0),((FJB0,GJB0),(NFA0,OFA0))))))),((((SCA0,TCA0),((CVA0,(OKB0,PKB0)),PRA0)),((ALA0,BLA0),(((GYA0,HYA0),OIA0),((YRA0,ZRA0),(((BDA0,((AFA0,(UQA0,VQA0)),(AXA0,(OGB0,PGB0)))),((FJA0,((WVA0,XVA0),(DUA0,EUA0))),(UKB0,VKB0))),KCA0))))),(((((WOA0,(NUA0,OUA0)),YNA0),((CBB0,DBB0),TOA0)),((NAB0,(MMB0,NMB0)),(JAA0,(PFA0,QFA0)))),((RQA0,SQA0),EFA0)))),((((((((((LLB0,MLB0),(LTA0,MTA0)),(ERA0,FRA0)),MGA0),(QTA0,RTA0)),((HGB0,IGB0),(VFA0,WFA0))),((DDA0,(QGB0,RGB0)),((CKB0,DKB0),WJA0))),((YHB0,ZHB0),(((GLA0,HLA0),(((OYA0,(RZA0,SZA0)),(PNA0,(DFB0,EFB0))),FHA0)),(KNA0,((GHB0,HHB0),KQA0))))),(((RWA0,SWA0),ADA0),ACA0)),(((((XOA0,(LPA0,(CWA0,DWA0))),(QKB0,RKB0)),(HBA0,(((LSA0,MSA0),QHA0),CFA0))),MAA0),(((LBA0,MBA0),XAA0),KAA0))))),((GIB0,HIB0),(DIA0,EIA0))),(((((((((((((JSA0,(WJB0,XJB0)),(PCB0,QCB0)),((JLA0,KLA0),((OBA0,(((FLB0,GLB0),TQA0),DCA0)),CBA0))),((GUA0,HUA0),(RCA0,(OMA0,PMA0)))),((PDA0,QDA0),(UGA0,(XBB0,YBB0)))),((IEA0,(IMA0,JMA0)),((KBA0,(VMA0,WMA0)),(LAA0,((((ZIA0,(NRA0,ORA0)),THA0),(EKA0,FKA0)),YAA0))))),(((UJB0,VJB0),BPA0),CCA0)),((((((CKA0,DKA0),((XPA0,YPA0),VEA0)),((QLA0,RLA0),(DNA0,(ESA0,FSA0)))),((((XEB0,YEB0),HWA0),(YVA0,ZVA0)),((PEB0,QEB0),JBA0))),((BEB0,CEB0),((KFA0,(EGB0,FGB0)),(YIA0,(DPA0,((NLB0,(WMB0,XMB0)),(WXA0,XXA0))))))),(EAA0,((YEA0,(XGA0,(FNA0,(TNA0,UNA0)))),(((ILA0,((MIB0,NIB0),WWA0)),(HNA0,INA0)),HCA0))))),(((QGA0,RGA0),(VDA0,((KWA0,LWA0),(IUA0,JUA0)))),((((DLB0,ELB0),(LRA0,MRA0)),((((PQA0,QQA0),(((HDB0,(QLB0,RLB0)),HCB0),(SXA0,(YAB0,ZAB0)))),ABA0),(SEA0,TEA0))),((((LJA0,MJA0),PEA0),((NFB0,OFB0),ZMA0)),(UHA0,(CIB0,DIB0)))))),(((VPA0,WPA0),PHA0),((((((HGA0,((CSA0,DSA0),(VDB0,(SKB0,TKB0)))),(HXA0,IXA0)),LDA0),((AJA0,BJA0),SFA0)),((XIB0,YIB0),IYA0)),(((HTA0,ITA0),(((FCB0,GCB0),(EVA0,(ZIB0,AJB0))),(VJA0,((KJB0,LJB0),LKA0)))),((YXA0,(YFB0,ZFB0)),(MXA0,NXA0)))))),(((((XKA0,(YOA0,ZOA0)),IHA0),((JPA0,KPA0),CLA0)),(YKA0,ZKA0)),(((NPA0,OPA0),KOA0),DBA0))),((((CAA0,(UDA0,(((OBB0,(OEB0,(LFB0,MFB0))),XRA0),((XZA0,YZA0),(FQA0,GQA0))))),((((XMA0,YMA0),((WAA0,(YHA0,((OWA0,PWA0),NMA0))),((((ULA0,(JAB0,(BCB0,CCB0))),(((VBB0,WBB0),BAB0),SVA0)),IBA0),((VHA0,(((TGB0,UGB0),KCB0),(GSA0,HSA0))),((GFA0,(GXA0,(TBB0,UBB0))),(((SMB0,TMB0),(RPA0,SPA0)),((((ODB0,PDB0),DYA0),(EOA0,FOA0)),((UFB0,VFB0),PYA0)))))))),((TAA0,((TFA0,UFA0),(ECA0,FCA0))),((TGA0,((BOA0,COA0),((NGB0,(AKB0,(KKB0,LKB0))),MLA0))),((KEB0,LEB0),(ZHA0,AIA0))))),((((((WQA0,XQA0),SGA0),((KAB0,LAB0),NLA0)),((((((KHB0,LHB0),(UAB0,VAB0)),(ZNA0,AOA0)),(RNA0,SNA0)),(ELA0,FLA0)),PCA0)),((CAB0,DAB0),(((UVA0,VVA0),MNA0),((RUA0,(MHB0,(YMB0,ZMB0))),APA0)))),(GEA0,HEA0)))),(((((((IDA0,((KMA0,LMA0),(CGA0,(IIA0,JIA0)))),(MWA0,NWA0)),(XFA0,YFA0)),((WKA0,(LLA0,(ANB0,BNB0))),JDA0)),(AHA0,BHA0)),((QYA0,RYA0),KDA0)),(((ZQA0,(VCB0,WCB0)),((TDA0,((JGA0,KGA0),(EBB0,FBB0))),ICA0)),((((VAA0,(((WSA0,(WZA0,(IKB0,JKB0))),((((RMA0,(HVA0,(GBB0,HBB0))),((SJB0,TJB0),UIB0)),((ANA0,BNA0),RJA0)),ZFA0)),(NEA0,(AHB0,BHB0)))),((EXA0,FXA0),(((JTA0,KTA0),BRA0),((CHA0,(HAB0,IAB0)),((((RKA0,(LOA0,(WUA0,XUA0))),(IOA0,JOA0)),XEA0),((((WGB0,XGB0),TSA0),EPA0),HFA0)))))),((((YKB0,ZKB0),SGB0),YLA0),SKA0)),((DGA0,EGA0),(VUA0,(NCB0,OCB0))))))),(((IVA0,JVA0),((URA0,VRA0),LGA0)),((((LHA0,MHA0),(UAA0,((((DEA0,(SMA0,(IRA0,JRA0))),((DVA0,(PFB0,QFB0)),GDA0)),((((GJA0,(BDB0,CDB0)),KHA0),(((YGB0,ZGB0),SDB0),(OLA0,PLA0))),((QNA0,(QDB0,RDB0)),((BKB0,(WKB0,XKB0)),(HPA0,IPA0))))),ZAA0))),((YUA0,ZUA0),QCA0)),((((UTA0,VTA0),(RFA0,((MBB0,NBB0),MMA0))),WCA0),((((EBA0,(HJA0,IJA0)),(((NHB0,OHB0),((DCB0,ECB0),CEA0)),(((TPA0,UPA0),VNA0),REA0))),(((((JFB0,KFB0),((XHB0,(SLB0,TLB0)),WRA0)),(((VZA0,(GAB0,(KMB0,LMB0))),(EAB0,FAB0)),CPA0)),(PPA0,QPA0)),((UIA0,VIA0),(PHB0,QHB0)))),(((DEB0,EEB0),KDB0),OKA0))))))),((LGB0,MGB0),RAA0)));
