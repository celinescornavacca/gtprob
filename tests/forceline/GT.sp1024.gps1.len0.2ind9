(((((((FDA0,((CIB0,(GNB0,HNB0)),(PMA0,QMA0))),((XLB0,YLB0),((PWA0,QWA0),WCA0))),((((LDA0,(((WAB0,XAB0),(BRA0,(BLB0,CLB0))),(IBB0,JBB0))),(GCA0,HCA0)),((TDA0,UDA0),(HJA0,IJA0))),((IMB0,JMB0),(GKB0,HKB0)))),(((UAA0,((((NEA0,OEA0),(HEA0,((KFB0,LFB0),(UHA0,(UJA0,VJA0))))),MCA0),ACA0)),(((((RLB0,SLB0),UBA0),(((PNA0,(BQA0,((KYA0,LYA0),((RHB0,SHB0),(MCB0,NCB0))))),(UGB0,VGB0)),(ODA0,(JZA0,(WFB0,XFB0))))),((YAB0,ZAB0),(SKA0,TKA0))),((MOA0,NOA0),((JFA0,KFA0),SDA0)))),((((MAB0,NAB0),((YMB0,ZMB0),(TLB0,ULB0))),(((ZDA0,(DCB0,ECB0)),((BMB0,CMB0),(((YGA0,((CBB0,DBB0),((TZA0,UZA0),RWA0))),((PRA0,(LWA0,MWA0)),(EMB0,FMB0))),LEA0))),((((QLA0,(FXA0,GXA0)),GBA0),(((ZSA0,ATA0),BIA0),(ONA0,(CHB0,DHB0)))),((TGA0,UGA0),VEA0)))),((((((EWA0,FWA0),(MIA0,NIA0)),(UTA0,VTA0)),((SMB0,TMB0),OFA0)),(((((OZA0,PZA0),LOA0),(BFA0,CFA0)),SCA0),(PBA0,QBA0))),(((((HFA0,IFA0),((UWA0,VWA0),WEA0)),(IAA0,((((IXA0,JXA0),GUA0),(((AWA0,BWA0),WFA0),(FFA0,GFA0))),(LMA0,((JKB0,KKB0),(AGB0,BGB0)))))),((((ITA0,((JJB0,KJB0),AJB0)),(((((QGB0,RGB0),FUA0),(UIB0,VIB0)),AQA0),((CGB0,DGB0),MSA0))),(MDA0,((ZFA0,((WIB0,XIB0),((OAB0,PAB0),QZA0))),((TFA0,(IGA0,JGA0)),(((KUA0,LUA0),((GAB0,HAB0),(SIB0,TIB0))),LHA0))))),(XEA0,YEA0))),(((HGA0,(HYA0,((PJB0,QJB0),GHB0))),(PDB0,QDB0)),JCA0)))))),(((PXA0,QXA0),XGA0),(((((QUA0,RUA0),(MQA0,NQA0)),BBA0),NAA0),((GBB0,HBB0),(MEA0,(((DUA0,EUA0),HRA0),(((FOA0,GOA0),AGA0),(LGA0,((TEB0,UEB0),(URA0,VRA0)))))))))),(((((((TMA0,UMA0),(((HHB0,IHB0),TQA0),XNA0)),ECA0),FAA0),((((KGA0,(QAB0,RAB0)),(((((IVA0,JVA0),(UCB0,VCB0)),(QDA0,(KQA0,LQA0))),((EBB0,FBB0),((LJB0,MJB0),JFB0))),(((WRA0,XRA0),(SGB0,TGB0)),(SJA0,(JOA0,KOA0))))),(CBA0,(((CEA0,((NRA0,ORA0),((XJB0,YJB0),UOA0))),(OXA0,((NHB0,OHB0),FZA0))),((SDB0,TDB0),(QGA0,RGA0))))),((WVA0,XVA0),QQA0))),(((QAA0,RAA0),((UXA0,VXA0),(VEB0,WEB0))),((((EGA0,(GGB0,HGB0)),((GVA0,HVA0),((XKB0,YKB0),(AEB0,(RKB0,SKB0))))),((((KIA0,LIA0),((OIB0,PIB0),((YWA0,(PHB0,QHB0)),(KBB0,(ZBB0,ACB0))))),(SPA0,TPA0)),(((((ZJB0,AKB0),(MYA0,NYA0)),((OMB0,PMB0),XHB0)),MMA0),(XCA0,(((CHA0,DHA0),SGA0),HDA0))))),(TCA0,UCA0)))),((LLB0,MLB0),(BTA0,(BDB0,CDB0))))),((((((OIA0,PIA0),(DKA0,((PYA0,(PLB0,QLB0)),(DPA0,EPA0)))),(NKA0,(VLA0,(((HLB0,ILB0),DIB0),OOA0)))),BAA0),(((((BJB0,CJB0),LBA0),(NJA0,OJA0)),((ZEA0,((IMA0,(ANA0,(((REB0,SEB0),VVA0),((((FLB0,GLB0),ODB0),XYA0),OVA0)))),((JRA0,KRA0),(QJA0,RJA0)))),((((NMA0,OMA0),PGA0),GGA0),YDA0))),(((((ZAA0,(ZCA0,((QOA0,(IPA0,JPA0)),(IRA0,(WCB0,(JEB0,KEB0)))))),((PHA0,(QIA0,(EJA0,(IQA0,JQA0)))),(ILA0,(XEB0,YEB0)))),(((((((DYA0,(OGB0,PGB0)),YJA0),(AHA0,BHA0)),(((BEB0,CEB0),((GQA0,HQA0),VDA0)),((AJA0,BJA0),SFA0))),(((CAB0,DAB0),((JWA0,KWA0),GRA0)),((VUA0,(BKB0,CKB0)),(TJB0,UJB0)))),((XBA0,(((MGB0,NGB0),KMA0),(WDB0,XDB0))),(((MPA0,NPA0),CLA0),(((VHB0,WHB0),(NWA0,OWA0)),JBA0)))),((((EYA0,FYA0),((JHB0,KHB0),SUA0)),(((BXA0,CXA0),(XFA0,YFA0)),(((PBB0,QBB0),QTA0),(MJA0,(SXA0,TXA0))))),(((KLA0,((ICB0,JCB0),BAB0)),(KCA0,((FDB0,(BFB0,CFB0)),SEA0))),(INA0,JNA0))))),(GAA0,(((MZA0,NZA0),ZPA0),(WMB0,XMB0)))),(((ZEB0,(JLB0,KLB0)),((((EDA0,((MMB0,NMB0),(RZA0,SZA0))),((GIB0,HIB0),EEA0)),((((((YNA0,ZNA0),(NBB0,OBB0)),(JIA0,(MNA0,(DRA0,ERA0)))),(IEA0,JEA0)),(((LKA0,MKA0),(VOA0,WOA0)),((OKA0,PKA0),((ADB0,(GDB0,HDB0)),ZBA0)))),(((QCB0,RCB0),((SAB0,TAB0),IHA0)),((CNA0,DNA0),((OCA0,(EHA0,(LKB0,MKB0))),OBA0))))),(((((((KAB0,LAB0),(HEB0,IEB0)),(((AMA0,BMA0),XHA0),((((SFB0,TFB0),DEB0),OYA0),((FLA0,(RQA0,SQA0)),(ENA0,FNA0))))),(((WGB0,XGB0),(UPA0,VPA0)),((TNA0,UNA0),(QIB0,RIB0)))),(IBA0,(((DKB0,EKB0),RXA0),WMA0))),(DAA0,((HBA0,(TBA0,(((VLB0,WLB0),NTA0),((EIB0,FIB0),(UMB0,VMB0))))),((YQA0,ZQA0),((GHA0,HHA0),AFA0))))),(((IKA0,(EMA0,FMA0)),(MBA0,((OTA0,PTA0),WPA0))),((((CSA0,DSA0),(MDB0,NDB0)),(LRA0,MRA0)),(WAA0,(BEA0,(GEA0,(((YYA0,ZYA0),PVA0),(RBB0,SBB0)))))))))),(((WDA0,XDA0),((RMA0,SMA0),KHA0)),((((TBB0,UBB0),UEA0),VAA0),((((((FKB0,(ZKB0,ALB0)),(RDB0,(YDB0,ZDB0))),JDA0),((((CRA0,(QVA0,(VZA0,WZA0))),((HJB0,IJB0),SYA0)),MFA0),((VJB0,WJB0),(APA0,(QSA0,RSA0))))),(CCA0,((UKA0,VKA0),NDA0))),(DBA0,((IKB0,(PKB0,QKB0)),((DGA0,(((YRA0,ZRA0),(ZCB0,(YFB0,ZFB0))),WHA0)),(BUA0,CUA0))))))))))),((((((((((AYA0,BYA0),KKA0),(AEA0,(HXA0,(IYA0,JYA0)))),(((GMB0,HMB0),IGB0),VBA0)),(((QHA0,RHA0),(GMA0,(ESA0,(KCB0,LCB0)))),((YHB0,ZHB0),((WXA0,XXA0),(NJB0,OJB0))))),((((KAA0,((QEA0,(VHA0,(MVA0,NVA0))),((IDB0,JDB0),((OQA0,PQA0),JKA0)))),((XJA0,((SCB0,TCB0),AZA0)),(KBA0,((AFB0,(NLB0,OLB0)),(TJA0,((DQA0,EQA0),(HOA0,IOA0))))))),(((HUA0,IUA0),NNA0),WBA0)),((((DLB0,ELB0),RFA0),(TOA0,(MFB0,NFB0))),(((((((AUA0,(WUA0,XUA0)),(MXA0,NXA0)),CDA0),((CQA0,(LEB0,MEB0)),(((INB0,JNB0),EIA0),FGA0))),(TAA0,(((JSA0,KSA0),(WIA0,XIA0)),KEA0))),SAA0),((DMB0,(CNB0,DNB0)),((UQA0,(VBB0,WBB0)),(PEB0,QEB0))))))),((LCA0,((BOA0,(WTA0,XTA0)),((IWA0,(XCB0,YCB0)),JHA0))),((((TEA0,(ENB0,FNB0)),(((AHB0,BHB0),(RVA0,SVA0)),(RDA0,((EFA0,(((FPA0,GPA0),GJA0),((WQA0,XQA0),((THB0,UHB0),FVA0)))),(BGA0,CGA0))))),((EBA0,FBA0),((ASA0,BSA0),(FCA0,((MIB0,NIB0),MGA0))))),(((TKB0,UKB0),(WWA0,XWA0)),((SWA0,(FJB0,GJB0)),FQA0))))),((((CTA0,DTA0),(LLA0,MLA0)),(((QRA0,RRA0),TLA0),((EKA0,(GKA0,HKA0)),(YIB0,ZIB0)))),(CAA0,((SHA0,THA0),((XSA0,YSA0),((FFB0,GFB0),BCA0)))))),((((((ROA0,SOA0),HMA0),((HSA0,ISA0),(PJA0,(((TUA0,UUA0),(LSA0,(VYA0,WYA0))),ULA0)))),((ABB0,BBB0),MTA0)),(((UVA0,(XBB0,YBB0)),(AVA0,BVA0)),((BPA0,CPA0),YIA0))),(((((RIA0,(GTA0,HTA0)),(((YHA0,(TVA0,(OCB0,PCB0))),(DLA0,ELA0)),((DFB0,EFB0),JTA0))),(JJA0,(KXA0,LXA0))),((LAA0,MAA0),(((((VNA0,WNA0),ZIA0),(YMA0,((UAB0,VAB0),ZMA0))),NBA0),(((ZLA0,((KIB0,LIB0),USA0)),(GLA0,HLA0)),ICA0)))),((((((LGB0,(AIB0,BIB0)),(TWA0,(ZWA0,AXA0))),(NCA0,((FRA0,((YGB0,ZGB0),OFB0)),((WKA0,XKA0),IIA0)))),((JAA0,((PFA0,(ETA0,FTA0)),((YOA0,ZOA0),BNA0))),(((GNA0,HNA0),YBA0),(TYA0,UYA0)))),((AKA0,BKA0),((((CJA0,DJA0),(HPA0,(DDB0,EDB0))),(EGB0,FGB0)),(VKB0,WKB0)))),(((COA0,DOA0),(UDB0,VDB0)),HAA0))))),((((((NGA0,(((CWA0,DWA0),(DXA0,EXA0)),AOA0)),(((ZHA0,(POA0,(QFB0,RFB0))),((ABA0,((((NLA0,OLA0),AIA0),(QKA0,RKA0)),QCA0)),(((ZKA0,ALA0),(RLA0,SLA0)),VCA0))),((SRA0,TRA0),((GCB0,HCB0),VMA0)))),((((SBA0,((((CYA0,(GZA0,HZA0)),XMA0),(MUA0,NUA0)),(PDA0,(NSA0,(RJB0,SJB0))))),(((XAA0,YAA0),(((((NKB0,OKB0),GEB0),FCB0),((YUA0,ZUA0),((YVA0,ZVA0),(FSA0,GSA0)))),GDA0)),((PFB0,(QMB0,RMB0)),FKA0))),((DDA0,((IIB0,JIB0),(GWA0,HWA0))),(((((XPA0,YPA0),(RTA0,STA0)),NFA0),((KVA0,LVA0),((QYA0,RYA0),(YTA0,(IZA0,(EHB0,FHB0)))))),(((VSA0,WSA0),FEA0),(QFA0,(BCB0,CCB0)))))),(ANB0,BNB0))),((WJA0,(LBB0,MBB0)),((((PLA0,(((GYA0,(XZA0,YZA0)),CVA0),JUA0)),((((DZA0,EZA0),QNA0),HIA0),(((IAB0,JAB0),YKA0),ZGA0))),(PEA0,(ZLB0,AMB0))),((((LHB0,MHB0),(UIA0,VIA0)),(((PCA0,(XLA0,YLA0)),((CKA0,((OSA0,PSA0),(UFB0,VFB0))),((EOA0,(HFB0,IFB0)),OGA0))),(((NHA0,OHA0),((YXA0,ZXA0),(((BZA0,CZA0),(OPA0,PPA0)),DFA0))),((DEA0,(VGA0,WGA0)),(((EAB0,FAB0),(((JGB0,KGB0),BLA0),(EEB0,FEB0))),RCA0))))),((OAA0,((FIA0,GIA0),REA0)),(((((NEB0,OEB0),SSA0),FJA0),YCA0),(KTA0,LTA0))))))),AAA0),(((((((FHA0,((ARA0,(ZZA0,AAB0)),WLA0)),(RNA0,SNA0)),((UFA0,VFA0),((JLA0,(TTA0,(KZA0,LZA0))),ZJA0))),(PAA0,(((((KMB0,LMB0),TSA0),VQA0),((KDB0,LDB0),(DJB0,EJB0))),RBA0))),(XOA0,(QPA0,RPA0))),(EAA0,((DCA0,(CMA0,DMA0)),(((OUA0,PUA0),ZTA0),BDA0)))),((((KNA0,LNA0),JMA0),(KDA0,(KJA0,LJA0))),(((CIA0,DIA0),((MHA0,(KPA0,LPA0)),LFA0)),((IDA0,((SIA0,TIA0),(DVA0,EVA0))),ADA0))))))));
