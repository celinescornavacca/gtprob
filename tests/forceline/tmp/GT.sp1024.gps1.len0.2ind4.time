43905.22user 16.10system 12:13:09elapsed 99%CPU (0avgtext+0avgdata 2576048maxresident)k
96inputs+8outputs (0major+3211292minor)pagefaults 0swaps
