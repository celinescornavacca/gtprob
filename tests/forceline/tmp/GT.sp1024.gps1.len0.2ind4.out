-1404.75370824
